src/Prooph/EventStore/Service/Stream/AggregateStreamStrategyFactory.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+ * This file is part of the prooph/proophessor.
+ * (c) 2014 - 2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 20.08.15 - 16:06
+ */
+
+namespace Prooph\EventStore\Service\Stream;
+
+use Interop\Container\ContainerInterface;
+use Prooph\EventStore\Stream\AggregateStreamStrategy;
+
+/**
+ * Class AggregateStreamStrategyFactory
+ * @package Prooph\EventStore\Service\Stream
+ */
+final class AggregateStreamStrategyFactory
+{
+    /**
+     * @param ContainerInterface $container
+     * @return AggregateStreamStrategy
+     */
+    public function __invoke(ContainerInterface $container)
+    {
+        $aggregateTypeStreamMap = [];
+
+        if ($container->has('configuration')) {
```

Comment:
In prooph/service-bus I used `config`. In ZF2 both service names are available. Do you know what is the actual service name in ZF2 and what is only the alias?
Maybe we should check what other frameworks use. `config` is shorter, but maybe to short.